README.md

```diff
@@ -57,12 +53,28 @@ Configure the SonataAdminBundle. e.g.:
             new Knp\Bundle\MenuBundle\KnpMenuBundle(),
             new Sonata\DoctrineORMAdminBundle\SonataDoctrineORMAdminBundle(),
             new Sonata\AdminBundle\SonataAdminBundle(),
-            
+            new Sonata\IntlBundle\SonataIntlBundle(),
+
+            // Blast
+            new Blast\OuterExtensionBundle\BlastOuterExtensionBundle(),
+            new Blast\CoreBundle\BlastCoreBundle(),
+            new lbr\BlastTestBundle\BlastTestBundle(),
+            new Blast\BaseEntitiesBundle\BlastBaseEntitiesBundle(),
+            new Blast\UtilsBundle\BlastUtilsBundle(),
+
+            // Attachments
+            new Librinfo\MediaBundle\LibrinfoMediaBundle(), 
+          
+            // Wisiwig editor
+            new Stfalcon\Bundle\TinymceBundle\StfalconTinymceBundle(),
+
             // your personal bundles
+            // ...
```

Comment:
As it is indicated in the installation of the bundle itself (on line 31th), it seems unnecessary to repeat it here.